can_messages.asciipb

```diff
@@ -293,17 +293,32 @@ msg {
 
 msg {
   id: 28
-  source: DRIVER_CONTROLS
-  # target: TELEMETRY
-  msg_name: "steering angle"
+  source: DRIVER_CONTROLS_STEERING
+  # target: DRIVER_CONTROLS_CENTER_CONSOLE
+  msg_name: "steering_output"
   can_data {
     u16 {
-      field_name_1: "steering_angle"
+      field_name_1: "control_stalk_analog_state"
+      field_name_2: "control_stalk_digital_state"
```

Comment:
Same comment here above for the analog inputs wrt your digital inputs.

If this is intended to be a bitset, please indicate by adding a suffix.

Otherwise, are you just assuming that only 1 digital input will be active at a time?